web/src/admin/blueprints/BlueprintListPage.ts

```diff
@@ -1,7 +1,6 @@
 import "@goauthentik/admin/blueprints/BlueprintForm";
```

Comment:
With the exception of the _addition_ of the `defaultEndpointConfig` definition [in Table.ts]( https://github.com/goauthentik/authentik/pull/10319/files#diff-7ae4e7a01898c3c9928af2e9c458a00be883a13867432c0b135c6ef770ac0168), everything is just deleting duplication; I don't have a copy of the script for this, sorry, I did it in elisp.